code/game/gamemodes/clock_cult/clock_scriptures/scripture_drivers.dm

```diff
@@ -153,52 +244,9 @@
 		animate(invoker.client, color = initial(invoker.client.color), time = 10)
 
 
-//Kindle: Charges the slab with blazing energy. It can be released to stun and silence a target.
-/datum/clockwork_scripture/ranged_ability/kindle
-	descname = "Short-Range Single-Target Stun"
-	name = "Kindle"
-	desc = "Charges your slab with divine energy, allowing you to overwhelm a target with Ratvar's light."
-	invocations = list("Divinity, show them your light!")
-	whispered = TRUE
-	channel_time = 30
-	power_cost = 125
-	usage_tip = "The light can be used from up to two tiles away. Damage taken will GREATLY REDUCE the stun's duration."
-	tier = SCRIPTURE_DRIVER
-	primary_component = GEIS_CAPACITOR
-	sort_priority = 6
-	slab_overlay = "volt"
-	ranged_type = /obj/effect/proc_holder/slab/kindle
-	ranged_message = "<span class='brass'><i>You charge the clockwork slab with divine energy.</i>\n\
-	<b>Left-click a target within melee range to stun!\n\
-	Click your slab to cancel.</b></span>"
-	timeout_time = 150
-	quickbind = TRUE
-	quickbind_desc = "Stuns and mutes a target from a short range."
-
-
-//Sigil of Submission: Creates a sigil of submission, which converts one heretic above it after a delay.
-/datum/clockwork_scripture/create_object/sigil_of_submission
-	descname = "Trap, Conversion"
-	name = "Sigil of Submission"
-	desc = "Places a luminous sigil that will convert any non-Servants that remain on it for 8 seconds."
-	invocations = list("Divinity, enlighten...", "...those who trespass here!")
-	channel_time = 60
-	power_cost = 125
-	whispered = TRUE
-	object_path = /obj/effect/clockwork/sigil/submission
-	creator_message = "<span class='brass'>A luminous sigil appears below you. Any non-Servants to cross it will be converted after 8 seconds if they do not move.</span>"
-	usage_tip = "This is the primary conversion method, though it will not penetrate mindshield implants."
-	tier = SCRIPTURE_DRIVER
-	one_per_tile = TRUE
-	primary_component = GEIS_CAPACITOR
-	sort_priority = 6
-	quickbind = TRUE
-	quickbind_desc = "Creates a Sigil of Submission, which will convert non-Servants that remain on it."
-
-
 //Replicant: Creates a new clockwork slab.
 /datum/clockwork_scripture/create_object/replicant
-	descname = "New Clockwork Slab"
+	descname = "New Clockwork Slab - Important!"
```

Comment:
Important!

is that really needed?

Reply:
It is; from my own experience people are braindead and constantly ask how to make new slabs, so with luck this'll make people do it more.